### ISSUE TITLE
Problem: The inline math inside item's title can not be properly exported

### ISSUE THREAD

**Issue by wangfh5** [retorquere/zotero-better-bibtex#2153]
> Support log ID: 2E5BNP4Q-refs-apse

I have a zotero item whose title contains inline math: 
```
Exact Solution for an $N$-Molecule---Radiation-Field Hamiltonian
```
After exporting it as a Better Biblatex `.bib` file: 
![image](https://user-images.githubusercontent.com/51868839/168467922-d2face43-6f1d-4d07-aaec-8b5d9a3aac12.png)
I got the following item: 
```
@article{Phys.Rev.1968Tavis,
  title = {Exact {{Solution}} for an \${{N}}\$-{{Molecule---Radiation-Field Hamiltonian}}},
  ......
  }
```
And then I used `biblatex` to cite reference in LaTeX. However, this kind of math code ` \${{N}}\$` can not be identified as inline math of LaTeX, because it turned out that `\printbibliography` gave me: 
![image](https://user-images.githubusercontent.com/51868839/168468118-8b6f1d7e-2842-4206-a2c1-595924a37de4.png)

In fact, the right math code that can be identified by `biblatex` is `{$N$}`. I Just need to embrace the math code with {}, 
```
@article{Phys.Rev.1968Tavis,
  title = {Exact {{Solution}} for an {$N$}-{{Molecule---Radiation-Field Hamiltonian}}},
  ...
  }
 ``` 
which can be printed as: 
![image](https://user-images.githubusercontent.com/51868839/168468488-e80d1cfe-44ba-4f74-839c-af64e143ae97.png)

In short, I hope that BetterBibtex can change the way of exporting math codes in items' title, just turn `$xxx$` to `{$xxx$}`.

**Comment by retorquere** [comment]
> Exact Solution for an $N$-Molecule---Radiation-Field Hamiltonian

It contains *LaTeX* inline math. Zotero doesn't have math support, so these dollar signs are just dollar signs in Zotero, and that's how BBT interprets them by default. You can either manually tell BBT to interpret this part as plain TeX by having

```
Exact Solution for an <script>$N$</script>-Molecule---Radiation-Field Hamiltonian
```

or have a postscript do it for you: https://retorque.re/zotero-better-bibtex/exporting/scripting/#detect-and-protect-latex-math-formulas

**Comment by wangfh5** [comment]
Thank you very much for your timely reply. The postscript you post works perfectly for me.
